api/test/integration/test_authorization_contexts_login_method.tavern.yaml

```diff
@@ -0,0 +1,66 @@
+---
+test_name: POST /security/user/authenticate/run_as
+
+marks:
+  - base_tests
+
+stages:
+
+  - name: Attempts to obtain the permissions associated with the default rule of the wazuh-wui user with a different user
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/security/user/authenticate/run_as"
+      headers:
+        Authorization: "Basic {basic_auth:s}"
+      method: POST
+      json:
+        username: "elastic"
+    response:
+      status_code: 200
+      verify_response_with:
+        function: tavern_utils:test_validate_auth_context
+        extra_kwargs:
+          expected_roles: []
+
+
+  - name: Login with an authorization context an a valid user
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/security/user/authenticate/run_as"
+      headers:
+        Authorization: "Basic {basic_auth:s}"
```

Comment:
We are using the same `user:pass` as above. Is this correct? Take a look at the var `basic_auth` in `common.yaml`